packages/visualizations/src/components/Table/Table.svelte

```diff
@@ -0,0 +1,103 @@
+<script lang="ts">
+    import type { Async } from '../../types';
+    import type { TableData, TableOptions } from './types';
+
+    import { generateId } from '../utils';
+
+    import SourceLink from '../utils/SourceLink.svelte';
+    import Header from './Header.svelte';
+    import Body from './Body.svelte';
+    import parseTheme from './utils/theme';
+
+    export let data: Async<TableData>;
+    export let options: TableOptions;
+
+    const tableId = `table-${generateId()}`;
+
+    $: ({ value: records } = data);
+    // FIXME: Eslint is in conflict with prettier
+    // eslint-disable-next-line object-curly-newline
+    $: ({ columns, title, subtitle, description, source, theme } = options);
+
+    $: style = parseTheme(theme);
+</script>
+
+<div class="container" {style}>
+    <div class="header">
+        {#if title}
+            <h3>{title}</h3>
+        {/if}
+        {#if subtitle}
+            <p>{subtitle}</p>
+        {/if}
+    </div>
+    <div class="table-wrapper">
```

Comment:
Could be applied directly on `<table>` with `border-collapse: seperate;`

Reply:
`.table-wrapper` is to control the scroll behavior. 
I'll try to use the border styles only the table tag

Reply:
Ho, I see. Maybe `display: block; overflow-x: auto` then (I didn't test). But the would avoid the small blank space between border and table when scrolling to far (it comes back, but can "stretch" a bit too far).

Reply:
I found  `overscroll-behavior-x: none;`to prevent the 'stretch'